emcc.py

```diff
@@ -1767,6 +1767,12 @@ def default_setting(name, new_default):
     settings.FETCH = 1
     settings.JS_LIBRARIES.append((0, 'library_asmfs.js'))
 
+  if settings.WASMFS:
+    state.forced_stdlibs.append('libwasmfs')
+    settings.FILESYSTEM = 0
+    settings.SYSCALLS_REQUIRE_FILESYSTEM = 0
+    # settings.JS_LIBRARIES.append((0, 'library_wasmfs.js'))
```

Comment:
If we decide not to include `library_wasmfs.js` in this PR, then this could be a TODO comment mentioning that it should be added in the future instead.